iceberg/iceberg-handler/src/main/java/org/apache/iceberg/mr/hive/IcebergTableUtil.java

```diff
@@ -328,4 +334,40 @@ public static PartitionData toPartitionData(StructLike key, Types.StructType key
     }
     return data;
   }
+
+  /**
+   * Generates {@link JobContext}s for the OutputCommitter for the specific table.
+   * @param configuration The configuration used for as a base of the JobConf
+   * @param tableName The name of the table we are planning to commit
+   * @param branchName the name of the branch
+   * @return The generated Optional JobContext list or empty if not presents.
+   */
+  static List<JobContext> generateJobContext(Configuration configuration, String tableName,
```

Comment:
i don't think it's a proper place for this, it should be part of `OutputCommitter`

Reply:
Done.